package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@colony/eslint-config-colony",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
Please use `npm version minor`

Reply:
Gotcha. As discussed, let's do this on master for a new release :+1: